start_session.php

```diff
@@ -126,6 +126,25 @@
             ]
         ],
     ],
+    'irmatube_premium' => [
+        'nextSession' => [
+            'url' => 'https://privacybydesign.foundation/demo/get_session_request.php'
+        ],
+        'request' => [
+            '@context' => 'https://irma.app/ld/request/disclosure/v2',
+            'disclose' => [[[
+                'pbdf.gemeente.personalData.fullname',
+                ]]],
```

Comment:
* Nitpick: incorrect indentation, and inconsistent placement of brackets wrt the other session requests.
* Since we don't really care about the actual value of the name in this demo, and since most non-dutch people won't be able to load this attribute, you might consider accepting also the name attributes of Facebook, Twitter or Linkedin using condiscon.

Reply:
added facebook as used in other demos